common/os_calls.c

```diff
@@ -2437,6 +2437,13 @@ g_strncasecmp(const char *c1, const char *c2, int len)
 }
 
 /*****************************************************************************/
+const char* APP_CC
+g_strstr(const char* c1, const char* c2)
```

Comment:
`*` sticks to the right, not to the left. Please see coding_style.md. The preexisting code is not always a good example.